### PR TITLE
chore: Update call-publication-builder.yaml and CODEOWNERS

### PR DESCRIPTION
**Pull request by themr0c**
## What does this pull request change

Call publication builder on changes to the master branch.

## Pull request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Review by nickboldt — APPROVED** [review]

**Review by l0rd — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # Global Owners
-* @rkratky @themr0c @max-cx @tstastna @eclipse/ecd-che-docs-reviewers
+* @rkratky @themr0c @max-cx @deerskindoll  @eclipse-che/ecd-che-docs-reviewers
```
Comment (benoitf) [inline .github/CODEOWNERS]:
```suggestion
* @rkratky @themr0c @max-cx @deerskindoll @eclipse-che/ecd-che-docs-reviewers
```

**Review by benoitf — COMMENTED** [review]
FYI it seems the ecd-che-docs-reviewers team is still on eclipse org, not eclipse-che org

https://github.com/orgs/eclipse/teams/ecd-che-docs-reviewers

**Comment by themr0c** [comment]
@benoitf for some reason, even if the group still exists, the CODEOWNERS file reports errors (and now @rkratky is the only person able to validate a pull request opened by myself...). I believe migrating the group is the safest option. I opened a request:  https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/1043

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,3 +1,2 @@
 # Global Owners
-# @max-cx and @tstastna are not part of the eclipse-che org (yet?)
-* @rkratky @themr0c @eclipse-che/ecd-che-docs-reviewers
+* @rkratky @themr0c @max-cx @deerskindoll @eclipse-che/ecd-che-docs-reviewers
```
Comment (themr0c) [inline .github/CODEOWNERS]:
```suggestion
* @rkratky @themr0c @eclipse-che/ecd-che-docs-reviewers
```

**Pull request merged**